src/main/java/models/Game.java

```diff
@@ -23,11 +25,23 @@ public Game(){
     }
 
     public void buildDeck() {
-        for(int i = 2; i < 15; i++){
-            deck.add(new Card(i,Suit.Clubs));
-            deck.add(new Card(i,Suit.Hearts));
-            deck.add(new Card(i,Suit.Diamonds));
-            deck.add(new Card(i,Suit.Spades));
+        if (isAmerican == true)
+        {
+            for (int i = 2; i < 15; i++){
+                deck.add(new Card(i,Suit.Clubs));
+                deck.add(new Card(i,Suit.Hearts));
+                deck.add(new Card(i,Suit.Diamonds));
+                deck.add(new Card(i,Suit.Spades));
+            }
+        }
+        else if (isAmerican == false)
+        {
+            for (int i = 1; i < 11; i++){
```

Comment:
The difference in the conditions for the for loop are part of handling the different number of cards, correct?